aoc/cmd/root.go

```diff
@@ -0,0 +1,105 @@
+package cmd
+
+import (
+	"aoc/pkg/advent"
```

Comment:
Nitpik: order these similar to python, built ins then external and then internal

Reply:
Goland just likes to order them alphabetically, it seems. :shrug: maybe there's some lint config?

Reply:
Leave an empty line between those blocks. It would order each block separately.